### DISCUSSION TITLE
npx kitql

### DISCUSSION THREAD

**Discussion by spences10** [jycouet/kitql#105] [Ideas]
A script to scaffold out the folders and files needed to get started with KitQL

- folders > lib/graphql/_kitql
- files > .graphqlrc.yaml, kitQlClient.ts
- codemod > add witch and run config to the `svelte.config.js` file

**Comment by jycouet** [comment]
I like the idea a lot. Thx for sharing.
Do you want some options as well? on/off things?

**Reply by spences10** [reply]
I think that just covers the bare minimum needed, the suggestions are all required, right?

I'm not sure what else could be an on/off option? I'm probably wrong though so will go with your suggestions 😊
